File: Modul 6/Soal 3/PRAK603-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int n1, n2;
    printf("Masukkan jumlah elemen baris pertama dan kedua (dipisahkan spasi): ");
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
        return 0;
    }

    int baris1[n1], baris2[n2];
    printf("Masukkan elemen baris pertama (dipisahkan spasi): ");
    for (int i = 0; i < n1; i++) {
        scanf("%d", &baris1[i]);
    }

    printf("Masukkan elemen baris kedua (dipisahkan spasi): ");
    for (int i = 0; i < n2; i++) {
        scanf("%d", &baris2[i]);
    }

    printf("Hasil Perkalian: ");
    for (int i = 0; i < n1; i++) {
        printf("%d ", baris1[i] * baris2[i]);
    }
    printf("\n");

    return 0;
}

File: Modul 6/Soal 3/PRAK603-2410817210014-ZAVIER GHANI.py
n1, n2 = map(int, input("Masukkan jumlah elemen baris pertama dan kedua (dipisahkan spasi): ").split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    baris1 = list(map(int, input("Masukkan elemen baris pertama (dipisahkan spasi): ").split()))
    baris2 = list(map(int, input("Masukkan elemen baris kedua (dipisahkan spasi): ").split()))
    
    hasil = [baris1[i] * baris2[i] for i in range(n1)]
    print("Hasil Perkalian:", *hasil)
